openmdao/utils/numba.py

```diff
@@ -0,0 +1,25 @@
+"""
+Optionally import numba and provide a dummy jit decorator if it is not available.
+
+If numba is not available, this module provides a dummy jit decorator that simply returns the
+original function.
+"""
+try:
+    import numba
+    from numba import *
+except ImportError:
+    numba = None
+    prange = range
+
+    # If numba is not available, just write a dummy jit wrapper.
+    # Code will still run at a significant performance hit.
+    def jit(f, *args, **kwargs):
```

Comment:
I think this will confuse the autodoc tools. They'll pick up the documentation for jit rather than the wrapped function.  The solution is to use `functools.wraps` like here:

https://github.com/OpenMDAO/dymos/blob/8cb3c2666b363287eb23d2b79001e8ce3ebcd2f0/dymos/transcriptions/explicit_shooting/barycentric_control_interp_comp.py#L12

Reply:
I thought functools.wraps was only necessary if the function was actually being wrapped, and it's not in this case.  The original function is just returned by the decorator, and that should be happening at class load time which I would think would be before the autodoc stuff even runs.